config/locales/en.yml

```diff
@@ -134,7 +134,7 @@ en:
     application_feedback: How can we improve %{section} section?
     application_feedback_thank_you: Thank you for your feedback
     referee:
-      refuse_feedback: Are you sure you will not give %{full_name} a reference?
+      refuse_feedback: You’ve declined to give a reference
```

Comment:
Asked about this on Slack; not sure you have actually declined at this point, so we may want to make the title active instead, something like ‘Decline to give a reference’ or similar